volatility3/framework/plugins/windows/info.py

```diff
@@ -3,15 +3,35 @@
 #
 
 import time
+import struct
 from typing import List, Tuple, Iterable
 
 from volatility3.framework import constants, interfaces, layers, symbols
 from volatility3.framework.configuration import requirements
-from volatility3.framework.interfaces import plugins
+from volatility3.framework.interfaces import plugins, configuration
 from volatility3.framework.renderers import TreeGrid
 from volatility3.framework.symbols import intermed
 from volatility3.framework.symbols.windows import extensions
+from volatility3.framework.layers import physical
 
+def rol(value: int, count: int, max_bits: int = 64) -> int: 
+    """A rotate-left instruction in Python"""
+    max_bits_mask = (1 << max_bits) - 1
+    return (value << count % max_bits) & max_bits_mask |  \
+           ((value & max_bits_mask) >> (max_bits - (count % max_bits)))
+
+def bswap_32(value: int):
```

Comment:
It would be nice if there is unity with `bswap_64`. :)

```suggestion
def bswap_32(value: int) -> int:
```

Reply:
This review may not be valid because the maintainer has proposed integration of functions below.

Reply:
added return type